pkg/webui/frontend/src/components/ConfigForm.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div>
+    <FormulateForm
+    name="config"
+    v-model="config"
+    @submit="handleSubmit"
+    >
+  <div class="inputs">
+      <div>
+        <FormulateInput
+          type="text"
+          label="Name"
+          validation="required"
+          validation-name="Name"
+          error-behavior="live"
+          :placeholder="placeholders.name"
+          v-model="config.name"
+        />
+        <FormulateInput
+          type="text"
+          label="Build Command"
+          validation="required"
+          validation-name="Build Command"
+          error-behavior="live"
+          :placeholder="placeholders.buildCommand"
+          v-model="config.buildCommand"
+        />       
+        <FormulateInput
+          type="text"
+          label="Run Command"
+          validation="required"
+          validation-name="Run Command"
+          error-behavior="live"
+          :placeholder="placeholders.runCommand"
+          v-model="config.runCommand"
+        />       
+         <FormulateInput
+          type="group"
+          name="environment"
+          :repeatable="true"
+          label="Environment"
+          add-label="+ Add Environment Variable"
```

Comment:
You could put labels in the consts file as well.
A good practice is to have a lang.js file that exports an object with all the texts you need.
You could do like:
```
export default {
    placeholders: {
        ...
    },
    labels: {
        ....
    }
}
```

This opens a way to have multilanguage support, you simply create file per language. lang.es.js, lang.he.js etc..